packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -33,20 +42,86 @@ export const prepareTransactionFeedKit = zone => {
   /** @type {PublishKit<CctpTxEvidence>} */
   const { publisher, subscriber } = makeDurablePublishKit();
 
-  return zone.exoClassKit('Fast USDC Feed', TransactionFeedKitI, () => ({}), {
-    admin: {
-      /** @param {CctpTxEvidence } evidence */
-      submitEvidence: evidence => {
-        trace('TEMPORARY: Add evidence:', evidence);
-        // TODO decentralize
-        // TODO validate that it's valid to publish
-        publisher.publish(evidence);
-      },
-    },
-    public: {
-      getEvidenceStream: () => subscriber,
+  const makeNoopInvitation = defineNoopInvitation(zcf, 'submitting evidence');
+
+  const makeTransactionOracleKit = prepareTransactionOracleKit(zone, {
+    handleEvidence: (oracleId, evidence) => {
+      trace('handleEvidence', oracleId, evidence);
     },
+    makeNoopInvitation,
   });
+
+  return zone.exoClassKit(
+    'Fast USDC Feed',
+    TransactionFeedKitI,
+    () => {
+      const oracles = zone.mapStore('oracles', {
```

Comment:
I believe `oracle` is the name for the service itself, so consider `operators` or `oracleOperators`. This might also mean we change verbs like `makeOracleInvitation` -> `makeOperatorInvitation`

Reply:
thanks, this ambiguity didn't sit well with me but I didn't think of a good alternative. operator is more accurate.